website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -216,22 +213,128 @@ get customSortingMethodForStatus() {
   };
 }
 
-// we define a callback function that listens to the `onSort` event in the table, and updates the tracked sort order values accordingly
+// we define a callback function that listens to the `onSort` event in the table,
+// and updates the tracked sort order values accordingly
 @action
 customOnSort(_sortBy, sortOrder) {
   this.customSortOrderForStatus = sortOrder;
 }
 ```
 
+### Density
+
+To create a condensed or spacious table, add `@density` to the table's invocation. Note that it only affects the table body, not the table header.
+
+```handlebars{data-execute=false}
+<Hds::Table
+  @model={{this.model.data}}
+  @columns={{array
+    (hash key="artist" label="Artist" isSortable=true)
+    (hash key="album" label="Album" isSortable=true)
+    (hash key="year" label="Release Year")
+  }}
+  @density="short"
+>
+  <:body as |B|>
+    <B.Tr>
+      <B.Td>{{B.data.artist}}</B.Td>
+      <B.Td>{{B.data.album}}</B.Td>
+      <B.Td>{{B.data.year}}</B.Td>
+    </B.Tr>
+  </:body>
+</Hds::Table>
+```
+
+### Alignment
+
+#### Vertical alignment
+
+To indicate that the table's content should have a middle vertical-align, use `@valign` in the table's invocation.
+
+!!! Info
+
+Note that vertical-align only applies to inline, inline-block and table-cell elements: you can't use it to vertically align block-level elements ([see MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align)).
+
+If you have more than just text content in the table cell, you'll want to wrap that content in a flex box and style accordingly.
```

Comment:
This content feels more related to the heading and example below than it does to the informational message above. What about moving it under the heading "Vertical alignment with additional cell content"?

Reply:
The goal was to explain why vertical align alone won't work, and what the consumer should do instead. WDYT?

Reply:
Right, I understand that. What I'm suggesting is that the content within the info banner feels more related to the code example under the heading "Vertical alignment with additional cell content" than where it's placed now.

It feels like there's opportunity to move it here for a more natural flow of content:
<img width="879" alt="Screenshot 2023-03-14 at 1 15 16 PM" src="https://user-images.githubusercontent.com/8553306/225126104-b7b58df2-8a6e-4d22-9441-b5c0f75485f3.png">

Reply:
I see what you mean. I've updated it per your suggestion.